.eslintrc.js

```diff
@@ -21,15 +21,15 @@ module.exports = {
     WorkboxSW: false,
   },
   rules: {
-    'jsdoc/check-types': 2,
-    'jsdoc/newline-after-description': 2,
-    'max-len': [2, {
+    "jsdoc/check-types": 2,
+    "jsdoc/newline-after-description": 2,
+    "max-len": [2, {
       code: 80,
       tabWidth: 2,
       ignoreComments: true,
-      ignoreUrls: true,
-      ignorePattern: '^\\s*import',
-    }],
+      "ignorePattern": "^\\s*import",
```

Comment:
These don't need quotes (it's a JS file, not JSON).

And elsewhere where there are quotes, could you switch them back to single and not double?

Reply:
I ran eslint or the `.eslintrc.js` file (not sure why it wasn't already doing that?).